mscr-ui/src/common/components/fixed-button-footer/index.tsx

```diff
@@ -0,0 +1,84 @@
+import {Button as Sbutton} from 'suomifi-ui-components';
+import ConfirmModal from '@app/common/components/confirmation-modal';
+import * as React from 'react';
+
+export enum FooterTypes {
+  'CROSSWALK_EDITOR',
+  'CROSSWALK_METADATA',
+  'SCHEMA_METADATA',
+}
+
+export default function FixedButtonFooter(props: { performFooterActionCallback?: any; footerType: FooterTypes; isEditModeActive: boolean; isPublished?: boolean}) {
+  const [isEditModeActive, setEditModeActive] = React.useState<boolean>(false);
+  const [isSaveConfirmModalOpen, setSaveConfirmModalOpen] = React.useState<boolean>(false);
+  const [isPublishConfirmModalOpen, setPublishConfirmModalOpen] = React.useState<boolean>(false);
+
+  function performFooterAction(action: string) {
```

Comment:
All but one of these use props.performFooterActionCallback but you've only checked for its existence in 2. Were you meaning to check it in others or is it ok if the others don't have a callback?

Reply:
This might be already redundant. I'll leave it as it is since this footer is going to be removed soon.